### PR TITLE
[Meta] Add some editorconfig to help with style

### PR DESCRIPTION
**Pull request by Nyholm**
This will just make things easier.

### CODE REVIEW

File: .github/workflows/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[*.yml]
```
Comment (pimjansen) [inline .github/workflows/.editorconfig:1]: Can't we just put it in 1 central editorconfig file?

Comment (Nyholm) [reply .github/workflows/.editorconfig:1]: But I only want to to effect the workflow files. 

Im not sure I can do that from the root editorconfig.

Comment (pimjansen) [reply .github/workflows/.editorconfig:1]: If im correct you can just add it and the IDE will pick it up from the app docroot and parse all files like that. Can you give it a try?

Comment (Nyholm) [reply .github/workflows/.editorconfig:1]: Yeah, But I dont want to apply this rule to ALL yml files. Only yml files in `.github/workflows`

**Review by pimjansen — APPROVED** [review]

**Comment by pimjansen** [comment]
Thanks @Nyholm

**Comment by Nyholm** [comment]
Thank you for merging

**Pull request merged**